Fix nowait poll

miker985 opened the pull request:
**This is currently live in production**

This remediates the problem sireq was having.

In more detail, this improves logging and fixes a loop that wasn't working as intended. Inside the `looper` function was a nested while loop. Here's a simplified and annotated version

```python
    while not tp.terminate.isSet():         # while nothing has told me to turn off (main)
        while not tp.terminate.isSet():     # INNER
            tp.timestamp = time.time()
            result = tp.handle.fetch()      # fetch new data
            tp.count = tp.count + 1
            tp.status = '... status ...'
            if not result:                  # if there was no new data, go to WAIT.
                break                       # otherwise repeat INNER and continue fetching new data
        tp.terminate.wait(...)              # WAIT some number of seconds before polling again
```

The problem being that the auth log was so busy there was **always new data**.

This led to continuous fetching of new data well beyond what Duo's rate limit allows for.  Per Cole on duo support ticket 01567856 (forwarded to Kevin and Jonathan) the maximum API rate is in the range of 2-3 requests per minute and

> ... the requests are being received [from logger4] every 2-4 seconds.

These API limits are **per Duo application integration** which left sireq largely locked out because **it used to share an integration with this**. It no longer does.

miker985 committed:
Add non-failure logging to duo_watcher to make activity more visible in logs

miker985 committed:
Remove inner loop so that there is a delay between polling even when lots of data is arriving

miker985 committed:
Update credentials section in Readme.md

miker985 committed:
Standardize on README.md

counik commented:
It all looks good, but could you elaborate on your last comment?

> These API limits are per Duo application integration which left sireq largely locked out because it used to share an integration with this. It no longer does.

miker985 commented:
I had @chmc3 create another Application Integration and updated duo_watcher to use that. I saved the details is mosler and updated the duo_watcher README to refer to them in this PR

sireq has had no changes and still uses the established integration. This was easiest because sireq is ref-managed and duo_watcher's credentials are not managed at all.
